Language/Haskell/GhcMod/Logger.hs

```diff
@@ -33,6 +35,13 @@ data Log = Log [String] Builder
 
 newtype LogRef = LogRef (IORef Log)
 
+data GmPprEnv = GmPprEnv { rsDynFlags :: DynFlags
```

Comment:
Just a nit-girtty convention thing: I usually use the capital letters in the type/constructor name as the prefix of the record member names, so `gpe` would be preferred.